### PR TITLE
Nav bar

### PR DESCRIPTION
**Pull request by sehilyi**
Added a simple navigation bar on the top with links for GitHub and documentations.

Towards #235

### CODE REVIEW

File: src/demo/App.scss [hunk 1]
```diff
@@ -1,24 +1,50 @@
...
+        background: #F6F6F6;
+        .header {
+            width: 50%;
+            margin-left: 25%;
+            font-size: 18px;
+            text-align: left;
+            padding-left: 15px;
+            padding-right: 15px;
+            .viewconf-options {
+                margin-left: 10px;
```
Comment (keller-mark) [inline src/demo/App.scss:19]:
```suggestion
                margin-left: 10px;
                font-size: 14px;
```

File: src/demo/App.scss [hunk 2]
```diff
@@ -1,24 +1,50 @@
 .app {
     text-align: center;
+    .header-container{
+        font-family: 'Lato', sans-serif;
+        font-weight: bold;
```
Comment (keller-mark) [inline src/demo/App.scss:5]:
```suggestion
        font-weight: normal;
```

**Comment by keller-mark** [comment]
Can we let the `.container` element scroll on its own so that the nav bar does not move with it? I imagine that to do this, we will need to give the nav bar and container elements fixed heights

or `.nav-bar{ height: 70px; }` and `.container { height: calc(100% - 70%); }` and also give the `body { height: 100vh; }` and then add `.container { overflow-x: scroll; }`?
Screen Shot 2020-05-18 at 12 02 19 PM

**Comment by sehilyi** [comment]
Thanks for sharing the screenshot to me, I had noticed the bad alignment of components on the top when the width is too narrow. I've fixed this alignment issue and updated to support horizontal scrolling separately for the nav bar and the main view on the bottom. 

Screen Shot 2020-05-18 at 3 19 53 PM

There still is an issue w/ scrolling; the wrapper components on the left are still hidden. Since this issue likely to be no problem for our demo for ITCR, let's fix this issue within a separate PR.

**Pull request merged**